ArduCopter/land_detector.cpp

```diff
@@ -54,7 +54,8 @@ void Copter::update_land_detector()
 #endif
             set_land_complete(false);
         }
-    } else {
+    } else if (copter.standby_active == false) {
```

Comment:
we can make this one !stsandby_active (i.e. remove the "== false" part).  also no need for the .copter part.

Reply:
one thing to be careful about here is that land_detector_maybe could get stuck as true because the land_detector_count variable will not be being updated while standby_active.  I suspect it's OK.. it will probably clear soon after the standby_active becomes false but worth thinking about.

Another option might be to change the cascading ifs a bit to add a few lines like this:

```
} else if (standby_active) {
    // land detector will not run in standby mode
    land_detector_count = 0;
} else {
    // all the normal stuff
```

Reply:
Yep, I like it.